app/controllers/concerns/idv_step_concern.rb

```diff
@@ -45,29 +45,30 @@ def flow_path
   end
 
   def confirm_hybrid_handoff_needed
-    setup_for_redo if params[:redo]
-
-    if idv_session.skip_hybrid_handoff?
-      # We previously skipped hybrid handoff. Keep doing that.
-      idv_session.flow_path = 'standard'
+    if params[:redo]
+      idv_session.redo_document_capture = true
+    elsif idv_session.document_capture_complete?
+      redirect_to idv_ssn_url && return
```

Comment:
`&&` binds "tighter" than the word "and" so this is like saying:

```ruby
redirect_to(idv_ssn_url && return)
```

which means that `redirect_to` is never called because the whole thing returns

confirmed via ruby-parse
```
> bundle exec ruby-parse -e "redirect_to idv_ssn_url && return"
(send nil :redirect_to
  (and
    (send nil :idv_ssn_url)
    (return)))
```

I think it would be easier to do 2 lines
```suggestion
      redirect_to idv_ssn_url
      return
```

in other cases I've seen the keyword `and` here because it has a looser binding... but I think rubucop may prevent

```suggestion
      redirect_to idv_ssn_url and return
```
```
> bundle exec ruby-parse -e "redirect_to idv_ssn_url and return"
(and
  (send nil :redirect_to
    (send nil :idv_ssn_url))
  (return))
```